test/CMakeLists.txt

```diff
@@ -22,19 +23,44 @@ if(SPIRV_TOOLS_FOUND)
   endif()
 
   set(SPIRV_TOOLS_BINDIR "${SPIRV_TOOLS_PREFIX}/bin")
+elseif(SPIRV-Tools-tools_FOUND)
```

Comment:
What about `SPIRV-Tools_FOUND` ?

Reply:
This particular branch doesn't use anything from `SPIRV-Tools` package...